src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -47,6 +47,9 @@ internal class WinHttpHandler : HttpMessageHandler
     public class WinHttpHandler : HttpMessageHandler
 #endif
     {
+#if NET46
```

Comment:
Are we already using such a compilation constant in WinHttpHandler and compiling it multiple times?  It'd have been nice to avoid that, as we've tried to do throughout corefx, especially if it's purely about the definition of a single static property.